test/integration.test.js

```diff
@@ -18,7 +18,7 @@ test('fastify will response swagger csp', async (t) => {
 
   await fastify.register(fastifySwagger)
   await fastify.register(fastifySwaggerUi)
-  fastify.register(fastifyHelmet, instance => {
+  await fastify.register(fastifyHelmet, instance => {
```

Comment:
This is likely the only needed await. Can you verify?

Reply:
All of them are needed.

Reply:
How? I don't think so.